Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h

```diff
@@ -281,6 +282,36 @@ DECLARE_SOA_TABLE(pidTPCAl, "AOD", "pidTPCAl", //! Table of the TPC response wit
 
 #undef DEFINE_UNWRAP_NSIGMA_COLUMN
 
+namespace pidbayes
+{
+// Bayes probability
+DECLARE_SOA_COLUMN(BayesEl, bayesEl, float);                        //! Bayesian probability of having a Electron
```

Comment:
Could we consider also here a reduced format (similar to the binned nsigma)?

Reply:
Hi, sure! Good idea! (Thanks for already having a look btw!)